browser_history/browsers.py

```diff
@@ -200,6 +312,25 @@ def bookmarks_parser(self, bookmark_path):
         ]
         return date_bookmarks
 
+    def cookies_parser(self, cookie):
+        """Returns cookies of a single input for Chrome based browsers
+        The returned datetimes are timezone-aware with the local timezone set
+        by default
+
+        :param cookie : dictionary containing cookie details
+        :type cookie : dict
+        :return a tuple of cookie information
+        :rtype: list(tuple(:py:class: str, str, `datetime.datetime`, str, str, str))
+        """
+        cookie["expiry"] = datetime.datetime.utcfromtimestamp(
+            cookie["expiry"]
+        ).strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
This can fail sometimes when cookies have invalid expiry timestamp (I had this issue on 2 of my systems).

Reply:
I tried the general thing we did in SQL, it gave me dates from 1970 , I am not that familiar with SQL, Ill see if i can dig up something online

Reply:
Which general thing?

```sql
            datetime(
                visit_date/1000000, 'unixepoch', 'localtime'
            ) AS 'visit_time',
```
this?

Reply:
yuup

Reply:
weird. but i think you can ignore the ones it is failing for because the expiry values were like `337520452448` which seems invalid as a UTC timestamp.